src/fragments/start/getting-started/reactnative/getting-started-steps-ui.mdx

```diff
@@ -22,4 +22,12 @@ You will also need to install the pod dependencies for iOS:
 npx pod-install
 ```
 </Block>
-</BlockSwitcher>
\ No newline at end of file
+
+If you ejected from Expo, you would also need to install Cognito library: 
+
+```sh
+npm install amazon-cognito-identity-js
+npx pod-install
+```
+
```

Comment:
```suggestion
```

Since, customers will also need `amazon-cognito-identity-js` when building apps using EAS as per [Expo docs](https://docs.expo.dev/build-reference/migrating/#amazon-cognito-identity-js--is-required-if-you-use). We should probably suggest all expo users to install `amazon-cognito-identity-js`.

Reply:
@majirosstefan , In order to resolve the issue for our customers asap, I have committed the requested changes. The getting started guide now suggests all expo users to install `amazon-cognito-identity-js`. Thank you bringing this to our attention and contributing to the project.